### PR TITLE
Add quill to bundleSetup

### PR DESCRIPTION
**Pull request by robertSt7**

### CODE REVIEW

**Review by mattamon — APPROVED** [review]
LGTM!

**Comment by jdreesen** [comment]
Shouldn't this have been included in the `2024.4` release?

**Comment by brusch** [comment]
@jdreesen yeah, we missed that one, but for skeleton it's not that critical, since it's not getting updated for existing installation, because it's using `composer create-project` and there it's just using the latest release.

File: src/EventSubscriber/BundleSetupSubscriber.php [hunk 1]
```diff
@@ -37,5 +38,6 @@ public function bundleSetup(BundleSetupEvent $event): void
     {
         // add required PimcoreAdminBundle
         $event->addRequiredBundle('PimcoreAdminBundle', PimcoreAdminBundle::class, true);
+        $event->addInstallableBundle('PimcoreQuillBundle', PimcoreQuillBundle::class, true);
```
Comment (brusch) [inline src/EventSubscriber/BundleSetupSubscriber.php]:
```suggestion
        $event->addRequiredBundle('PimcoreQuillBundle', PimcoreQuillBundle::class, true);
```

Should we make it a required bundle? 
Probably every developer expects to have a WYSIWYG when installing Pimcore, right? 🤔

**Review by brusch — CHANGES_REQUESTED** [review]

**Pull request merged**